tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.h

```diff
@@ -196,11 +196,23 @@ union InnerArrayExplicit // size = 32 bytes
 #endif
 
 #ifndef WINDOWS
-	struct OUTER3 // size = 28 bytes
-	{
-		struct InnerSequential arr[2];
-		LPCSTR f4;
-	};
+    #ifdef __x86_64__
+        union OUTER3 // size = 32 bytes
```

Comment:
Fix the alignment here. Also if you are declaring a size in comments, instead please use the `static_assert()` to compute and make it enforceable.

Reply:
The alignment here is actually correct. The alignment of the `f4` field needs to be at the 24th byte on all architectures that this test runs on to match the managed definition (which gives the `f4` field an explicit `FieldOffset(24)`).

Reply:
Uh... sorry. By alignment I meant source code. Please do the `static_assert` for size.